sdk/core/azure-core-experimental/azure/core/experimental/transport/_httpx_async.py

```diff
@@ -49,9 +60,23 @@ def __init__(
         )
 
     def body(self) -> bytes:
+        """Return the whole body as bytes.
+
+        :return: The whole body as bytes.
+        :rtype: bytes
+        """
         return self.internal_response.content
 
     def stream_download(self, pipeline: Pipeline, **kwargs: Any) -> AsyncIterator[bytes]:
+        """Generator for streaming response data.
+
+        :param pipeline: The pipeline object
+        :type pipeline: ~azure.core.pipeline.Pipeline
+        :keyword bool decompress: If True which is default, will attempt to decode the body based
```

Comment:
You have ":keyword bool decompress: If True which is default, will attempt to decode the body based" in async stream_download but not sync one.

Let's make them consistent. (I am open to either way removing both or keeping both)

Reply:
great catch, it should be there